File: abc/abc130/a.java
// https://atcoder.jp/contests/abc130/submissions/41617890

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int x = stdIn.nextInt();
    int a = stdIn.nextInt();

    System.out.println(x < a ? 0 : 10);
  }
}

File: abc/abc130/a.cs
// https://atcoder.jp/contests/abc130/submissions/46913473

class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int x = int.Parse(t[0]);
    int a = int.Parse(t[1]);

    if(x < a)
    {
      Console.Write(0);
    }
    else
    {
      Console.Write(10);
    }
  }
}

